assets/js/serviceworker.js

```diff
@@ -0,0 +1,90 @@
+var CACHE_NAME = 'sass-guidelines-cache';
+// The files we want to cache
+var urlsToCache = [
+  'index.html',
+  'cz/index.html',
+  'da/index.html',
+  'de/index.html',
+  'es/index.html',
+  'fr/index.html',
+  'ko/index.html',
+  'pl/index.html',
+  'pt/index.html',
+  'ru/index.html',
+  'zh/index.html',
+  'css/full.css',
+  'css/fonts/data-ttf.css',
+  'css/fonts/data-woff.css',
+  'css/fonts/data-woff2.css',
+  'assets/js/main.min.js',
+  'assets/js/picturefill.min.js',
+  'assets/images/css-order-chart.png',
+  'assets/images/lighten-darken-mix_huge.png',
+  'assets/images/lighten-darken-mix_large.png',
+  'assets/images/lighten-darken-mix_medium.png',
+  'assets/images/lighten-darken-mix_small.png',
+  'assets/images/sass-wallpaper_huge.jpg',
+  'assets/images/sass-wallpaper_large.jpg',
+  'assets/images/sass-wallpaper_medium.jpg',
+  'assets/images/sass-wallpaper_small.jpg',
+  'assets/images/sassdoc-preview_huge.png',
+  'assets/images/sassdoc-preview_large.png',
+  'assets/images/sassdoc-preview_medium.png',
+  'assets/images/sassdoc-preview_small.png',
+  'favicon.ico'
+];
+
+// Set the callback for the install step
+self.addEventListener('install', function(event) {
+    // Perform install steps
+    // event.waitUntil(
+    caches.open(CACHE_NAME)
+      .then(function(cache) {
+        console.log('Opened cache');
+        return cache.addAll(urlsToCache);
```

Comment:
`addAll` right now may hive different behavior in different browsers. You either need `cache-polyfill` (https://github.com/dominiccooney/cache-polyfill)  or some kind of normalized function, see this for example: https://github.com/NekR/offline-plugin/blob/master/src/misc/sw-template.js#L357